reana-ui/src/pages/workflowList/components/WorkflowList.js

```diff
@@ -68,6 +73,16 @@ export default function WorkflowList({ workflows, loading }) {
                 <Icon name="hdd" />
                 {size}
               </span>
+              {isSessionOpen && (
+                <span
+                  onClick={(e) =>
+                    openJupyterNotebook(e, session_uri, reanaToken)
+                  }
+                  className={styles.notebook}
+                >
+                  <JupyterNotebookIcon />
+                </span>
```

Comment:
Food for thought: In the beginning, when we implemented the workflow list, it was a very simple read-only list of items with a single possible action: clicking on them to see their details. This was the reason why we opt for applying a hover effect highlighting the whole element, as no matter where you clicked, it would always trigger the same action, going to the details page. As this page started to grow, new actions were introduced with other clickable elements inside the item, and it's likely that more of them will come. This results in the highlighting not being so useful anymore yet it's becoming a bit confusing because if one miss-clicks on the actions hamburger menu or the jupyter notebook icon, they will be redirected to the details page => bad UX.

Therefore we should find some way to improve this, maybe we can keep the hover effect but not enable de cursor pointer and only go to details when clicking on the workflow name?  Add another element/icon/button to go to the details? Opinions welcome.

If you think this makes sense, we can open an issue from this message.

Reply:
Completely agree. We should definitely change this behavior since as you mentioned it leads to a bad UX. Created a new issue from your comment to address it: https://github.com/reanahub/reana-ui/issues/145